pkg/registry/registry/repository/storage/storage.go

```diff
@@ -158,6 +169,7 @@ func (r *REST) Update(ctx context.Context, name string, objInfo rest.UpdatedObje
 
 // Delete enforces life-cycle rules for cluster termination
 func (r *REST) Delete(ctx context.Context, name string, deleteValidation rest.ValidateObjectFunc, options *metav1.DeleteOptions) (runtime.Object, bool, error) {
+	usr, _ := authentication.UsernameAndTenantID(ctx)
```

Comment:
change `usr` to `userName` in all your files

Reply:
done